mlflow/R/mlflow/DESCRIPTION

```diff
@@ -24,11 +24,13 @@ Depends:
   R (>= 3.1.2)
 Imports:
   aws.s3,
+  caTools,
```

Comment:
The [caTools](https://cran.r-project.org/web/packages/caTools/index.html) package is orphaned, use [base64enc](https://CRAN.R-project.org/package=base64enc) instead.

Reply:
Good catch, thanks!